lib/portage/tests/ebuild/test_doebuild_fd_pipes.py

```diff
@@ -106,6 +119,10 @@ def testDoebuild(self):
             )
             settings.setcpv(pkg)
 
+            # Demonstrate that settings.configdict["pkg"]["USE"] contains our arbitrary
+            # package.use setting in order to trigger bug 675748.
+            self.assertEqual(settings.configdict["pkg"]["USE"], arbitrary_package_use)
+
```

Comment:
> Any chance of a test case, or is it a pain because of ResolverPlayground?

I was able to add some package.env and package.use settings in testDoebuild that create a state like the one which triggered bug 925625.

> Does it work if we do it with global USE? I guess not? If not, it's fine obviously and I wouldn't worry about it.

By global USE do you mean like configdict["env"]["USE"] which would be like the setting from emerge --info? That value should never have any effect on _validate_deps.

Reply:
I also realized that we could do this to avoid a redundant setcpv call in _validate_deps:
```python
    # A USE calculation from setcpv should always be available here because
    # mysettings.mycpv is not None, so use it to prevent redundant setcpv calls.
    metadata["USE"] = mysettings["PORTAGE_USE"]
```